IaC/AKS/README.md

```diff
@@ -330,6 +332,49 @@ kubectl create secret generic ngsa-aks-secrets \
 
 ```
 
+## Setup SSL/DNS
+
+### DNS Setup
+
+Assuming that you have a domain name, and permissions to update DNS records for your domain, create an A record to point your domain to `INGRESS_PIP`.
+
+For example, in Azure:
+
+```bash
+
+az network dns record-set a add-record -g <dns zone resource group> -z <dns zone name> -n $Ngsa_Name -a $INGRESS_PIP --ttl <TTL seconds>
+
+```
+
+Set the application endpoint
+
+```bash
+
+export Ngsa_App_Endpoint=replaceWithDNSHostname
+
+```
+
+### Install Cert-Manager
+
+```bash
+
+cd $REPO_ROOT/IaC/AKS/cluster/manifests/cert-manager
+
+kubectl create ns cert-manager
+helm install cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --version v1.0.3 \
```

Comment:
For the cert-manager version, I think it's best if we put the version in an environment variable (similar to what we did with `ISTIO_VERSION` and `Ngsa_K8S_VER` )